repositories.yaml

```diff
@@ -1,5 +1,18 @@
-glean:
-  app_id: glean
+glean-core:
+  app_id: glean-core
+  notification_emails:
+    - frank@mozilla.com
+    - mdroettboom@mozilla.com
+  url: 'https://github.com/mozilla/glean'
+  branch: main
+  metrics_files:
+    - 'glean-core/metrics.yaml'
+  ping_files:
+    - 'glean-core/pings.yaml'
+  library_names:
+    - org.mozilla.glean-core
```

Comment:
nit: is this required to be `org.mozilla.glean-core` (android specific?)? Can this be `glean-core` or `lib-glean-core`? We don't need to change it, just curious.

Reply:
Nope, it can be any valid string, really. Would you prefer `glean-core`?

Reply:
Slight preference for that, yes, which would give this a less Android-specific flavour.